### PR TITLE
Edited meta tags for basic SEO of the page 🎆

### PR DESCRIPTION
**Pull request by FireQueen-3010**
ISSUE #9 
1) Added short icons for all types of devices
2) Added manifest file 
3) Edited description and changed title 

image

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
Please review @ArunTeltia @LordShenron

**Review by LordShenron — APPROVED** [review]
Looks clean to me. +1 for this

File: public/index.html [hunk 1]
```diff
@@ -3,16 +3,23 @@
...
   <meta charset="utf-8" />
-  <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+  <link rel="icon" href="%PUBLIC_URL%/icons/favicon.ico" />
+  <link rel="icon" type="image/png" sizes="32x32" href="%PUBLIC_URL%/icons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="%PUBLIC_URL%/icons/favicon-16x16.png">
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
-  <meta name="description" content="Web site created using create-react-app" />
-  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <meta name="description" content=" ProjectSakura was started by LordShenron as a hobby Project in
```
Comment (ArunTeltia) [inline public/index.html:11]: Dont need our name in the description no one will search for our name that will result in project sakura write the description about the project 
check this 
"Project Sakura provides quick timely updates and more customizations with the same stability as Lineage OS."
please write this

**Review by ArunTeltia — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (FireQueen-3010) [reply public/index.html:11]: Okay

**Comment by FireQueen-3010** [comment]
Made the required changes, please review :) @ArunTeltia

**Review by ArunTeltia — APPROVED** [review]

**Pull request merged**